src/diffusers/models/unet_2d.py

```diff
@@ -30,7 +30,7 @@ def __init__(
         attention_head_dim=8,
         norm_num_groups=32,
         norm_eps=1e-5,
```

Comment:
```suggestion
        norm_eps: float =1e-5,
```

Reply:
Hello

From issue #287, I see that type hints are required for all the variables. So would you be adding the type hints for other variables as well? Or is this PR out of scope for all of them?

Here are some examples:
```
flip_sin_to_cos: bool=True,
down_block_types: tuple[str]=("DownBlock2D", "AttnDownBlock2D", "AttnDownBlock2D", "AttnDownBlock2D"),
block_out_channels: tuple[int]=(224, 448, 672, 896),
act_fn: str="silu"
```

Reply:
Thanks for the examples @vishnu-anirudh and @patrickvonplaten - I added type hints for other `__init__` variables as well.